src/main/java/uk/gov/justice/hmpps/prison/service/enteringandleaving/CaseNoteMovementService.kt

```diff
@@ -14,13 +14,13 @@ import uk.gov.justice.hmpps.prison.security.AuthenticationFacade
 import uk.gov.justice.hmpps.prison.service.EntityNotFoundException
 
 @Service
-class CaseNoteTransferService(
+class CaseNoteMovementService(
```

Comment:
As the package deals with general movements rather than just transfers a few supporting services have been renamed accordingly.